cypress/e2e/process.cy.js

```diff
@@ -0,0 +1,166 @@
+            processesPage.textAutomaticEligibilityChecksPassed().should('be.visible');
+            tenureReqDocsPage.selectYesFor12Months().should('exist');
+            tenureReqDocsPage.selectNoForOccupyanyOther().should('exist');
+            tenureReqDocsPage.selectYesForSurvivorOfOne().should('exist');
+            tenureReqDocsPage.selectYesForTenantEvicted().should('exist');
+            tenureReqDocsPage.selectYesForImmigrationControl().should('exist');
+            tenureReqDocsPage.selectYesForLiveNotice().should('exist');
+            tenureReqDocsPage.selectYesForRentArrears().should('exist');
+        })
+    })
+    it('should go back or cancel process', ()=> {
+        seedDatabase();
+        cy.getTenureFixture().then((tenureInfo) => {
+            processesPage.visit(tenureInfo.id)
+            processesPage.cancelProcessLink().click()
+            cy.getTenureFixture().then((tenureInfo) => {
+                cy.url().should("include", `processes/tenure/${tenureInfo.id}`)
+            })
+        })
+        cy.getTenureFixture().then((tenureInfo) => {
+            processesPage.visit(tenureInfo.id)
+            processesPage.backLink().click()
+            cy.getTenureFixture().then((tenureInfo) => {
+                cy.url().should("include", `processes/tenure/${tenureInfo.id}`)
+            })
+        })
+    })
+
+    it('should fail automatic checks and close process should be intiated', ()=> {
+        seedDatabase()
+        cy.getTenureFixture().then((tenureInfo) => {
+            processesPage.visit(tenureInfo.id)
+
+            cy.getAssetFixture().then((asset) => {
+                const { assetAddress } = asset;
+                processesPage.tenureDetails().should("be.visible")
+                processesPage.tenureDetails().contains(`Tenure payment ref ${asset.tenure.paymentReference}`)
+                processesPage.tenureDetails().contains(`${assetAddress.addressLine1} ${assetAddress.addressLine2} ${assetAddress.addressLine3} ${assetAddress.postCode}`)
+            })
+
+            processesPage.startProcessButton().should("be.disabled")
+            processesPage.agreementCheckBox().click()
+            processesPage.startProcessButton().click()
+            cy.url().should("include", "processes/soletojoint/")
+            cy.get(`#select-tenant-${tenureInfo.householdMembers[2].id}`).click({ force: true })
+            cy.contains("Next").click();
+            processesPage.textAutomaticChecksFailed().should('be.visible');
+            tenureReviewDocsPage.checkboxConfirmOutcomeLetter().click();
+            tenureReviewDocsPage.buttonConfirm().click();
+            cy.contains('Thank you for your confirmation');
+            cy.contains("This case is now closed and we have recorded this on the system - that you have sent an outcome letter to the resident. The outcome can be viewed in the activity history");
+            cy.contains("a", "Return to home page").should("have.attr", "href");

```

Comment:
It contains the `href`, however, asserting this against `a` tag element does tell much due to `href` being a mandatory attribute.

Plus, even when it's provided, it's sometimes merely set to `#`, which doesn't link to anywhere.

It may be more useful to assert against the value of `href` to see whether it links the home page url.